### ISSUE TITLE
bug: kdeconnect-indicator not visible in systray

### ISSUE THREAD

**Issue by addy419** [nix-community/home-manager#3120]
### Are you following the right branch?

- [X] My Nixpkgs and Home Manager versions are in sync

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue description

kdeconnect-indicator when started from systemd service does not show up in systray unless restarted. The systemd service is running fine and has no errors. I am using qtile and qtile bar. The indicator shows up and works normally when started manually from terminal.

another issue, it does not respect the system theme and defaults to the blueish color theme (I guess it's called oxygen). This works fine when starting kde-connect indicator as `kdeconnect-indicator &` from terminal. If it's relevant, I am using qtile through home manager (.xsession) and the themes are set through kvantum manager and lxqt-config

### Maintainer CC

@adisbladis 

### System information

```markdown
`nix-shell -p nix-info --run "nix-info -m"` did not run, maybe because my config is managed through flakes. Here's the output of `nix-info -m`

 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.55, NixOS, 22.05 (Quokka), 22.05.20220722.e3583ad`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.9.1`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```
```

**Comment by addy419** [comment]
It was an issue with pixbuf dependency not present

**Comment by ajgrf** [comment]
@addy419 Would you mind elaborating on what the issue was and how you fixed it? I have the same problem.

**Comment by nilp0inter** [comment]
@ajgrf I also experience the same issue. Not sure how to debug it, though :disappointed: . A workaround is working for me is to restart the user service like so:

```console
$ systemctl --user restart kdeconnect-indicator
```

**Comment by sersorrel** [comment]
I looked into this a while ago. in at least some cases (e.g. for me using i3wm) there is no appropriate dependency that kdeconnect-indicator.service can refer to so that it only starts up once the tray is actually ready for applications to put tray icons in it (so it ends up starting too early and doesn't appear to handle that happening at all). I ended up starting kdeconnect-indicator manually in my i3 configuration rather than relying on the systemd service.
